hoomd/hpmc/UpdaterQuickCompress.cc

```diff
@@ -48,13 +48,13 @@ void UpdaterQuickCompress::update(uint64_t timestep)
     auto n_overlaps = m_mc->countOverlaps(false);
     BoxDim current_box = m_pdata->getGlobalBox();
 
-    if (n_overlaps == 0 && current_box != *m_target_box)
+    if (n_overlaps == 0 && current_box != (*m_target_box)(timestep))
         {
         performBoxScale(timestep);
```

Comment:
```suggestion
        performBoxScale(timestep, target_box);
```
Add `const BoxDim& target_box` to the arguments of `performBoxScale`.